build.gradle.kts

```diff
@@ -225,6 +224,32 @@ swaggerSources {
     }
 }
 
+configure<com.diffplug.gradle.spotless.SpotlessExtension> {
+    format("misc") {
+        target("*.gradle", ".gitattributes", ".gitignore", ".containerignore", ".dockerignore")
+
+        trimTrailingWhitespace()
+        indentWithSpaces()
+        endWithNewline()
+    }
+    java {
+        targetExclude("**/src/generated/**", "**/build/generated/**")
+
+        googleJavaFormat().reflowLongStrings().skipJavadocFormatting()
+
+        removeUnusedImports()
+        importOrder("java", "javax", "jakarta", "com", "org", "net.atos")
+
+        formatAnnotations()
+        licenseHeader("""
+            /*
+             * SPDX-FileCopyrightText: 2022 Atos, 2023-2024 Lifely
```

Comment:
New files should not contain copyrights by "Atos". Existing files should keep header as is.

Reply:
Unless touched by us ofc.

Reply:
True.. That makes this quite tricky I think..

Reply:
Lifely-only license files are updated. The rest are left alone and needs our attention.